README.md

```diff
@@ -64,7 +64,9 @@ apiGateway.stats.getTotalUsers()
   * [`getNewfilesBySiteAndCategory(params)`](https://build.envato.com/api/#!/market/NewFiles)
   * [`findFeaturedItemsBySite(site)`](https://build.envato.com/api/#market_Features)
   * [`randomNewItemsBySite(params)`](https://build.envato.com/api/#market_RandomNewFiles)
-* _coming soon_
+* recommender
+  * `getRecommendedItems(itemId)`
+  * `getRecommendedSearches(term, site)`
```

Comment:
Sure! I guess this pull request should only be merged after the documentation is published, otherwise does not make much sense to add to this package.